sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/models/options/UnassignJobOptions.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.jobrouter.models.options;
+
+/**
+ * Request options to unassign a job from a worker.
+ */
+public final class UnassignJobOptions {
+    /**
+     * id of job to close.
+     */
+    private final String jobId;
+
+    /**
+     * id of assignment associated with the job.
+     */
+    private final String assignmentId;
+
+    /**
+     * If set to true, the service will not re-match the job to a worker.
+     */
+    private Boolean isWorkerRematchPending;
```

Comment:
Let's not add this field right away. Same for getter and setter

Reply:
Done.